packages/dx-vue-grid/src/plugins/editing-state.js

```diff
@@ -0,0 +1,176 @@
+import { Getter, Action, Plugin } from '@devexpress/dx-vue-core';
+import {
+  createRowChangeGetter,
+  startEditRows,
+  stopEditRows,
+  addRow,
+  changeAddedRow,
+  cancelAddedRows,
+  addedRowsByIds,
+  changeRow,
+  cancelChanges,
+  changedRowsByIds,
+  deleteRows,
+  cancelDeletedRows,
+  getColumnExtensionValueGetter,
+} from '@devexpress/dx-grid-core';
+
+const columnExtensionValueGetter = (columnExtensions, defaultValue) =>
+  getColumnExtensionValueGetter(columnExtensions, 'editingEnabled', defaultValue);
+
+export const EditingState = {
+  name: 'EditingState',
+  props: {
+    createRowChange: {
+      type: Function,
+    },
+    columnEditingEnabled: {
+      type: Boolean,
+      default: true,
+    },
+    columnExtensions: {
+      type: Array,
+      default: undefined,
```

Comment:
I'm not sure we should specify `undefined` as default prop value. It's `undefined` by default.